Clinical-Genomics/scout
# Fix #2033 -- clinical filter returning no variants

Fix #2033.

**How to test**:
1. Go to scout stage and install the current master branch.
1. Open a case and go to variants page
1. Click on the clinical filter button: 

![image](https://user-images.githubusercontent.com/28093618/91155229-d9c79100-e6c2-11ea-83e6-ea97f145fba6.png)
1. No variants should be returned
1. Install this branch on scout stage
1. Repeat the search and variants should be shown

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
